docs/guides/rate-limits-project-based.mdx

```diff
@@ -0,0 +1,100 @@
+5. `*`: All other paths
+
+## Rate Limit Tables by Plan
+
+### Developer Plan Rate Limits
+
+| Environment   | Path / Bucket              | Burst (rps) | Sustained (rpm) |
+| :------------ | :------------------------- | ----------: | --------------: |
+| **Developer** | `/sessions/whoami`         |          10 |             300 |
+|               | `/admin/oauth2/introspect` |          10 |             300 |
+|               | `/relation-tuples/check`   |          10 |             300 |
+|               | `GET /admin/identities`    |           1 |              10 |
+|               | `*`                        |           5 |             150 |
+
+### Production Plan Rate Limits
+
+| Environment    | Path / Bucket              | Burst (rps) | Sustained (rpm) |
+| :------------- | :------------------------- | ----------: | --------------: |
+| **Production** | `/sessions/whoami`         |          80 |            1800 |
+|                | `/admin/oauth2/introspect` |          80 |            1800 |
+|                | `/relation-tuples/check`   |          80 |            1800 |
+|                | `GET /admin/identities`    |          10 |             300 |
+|                | `*`                        |          40 |             900 |
+| **Staging**    | `/sessions/whoami`         |          10 |              30 |
+|                | `/admin/oauth2/introspect` |          10 |             300 |
+|                | `/relation-tuples/check`   |          10 |             300 |
+|                | `GET /admin/identities`    |           1 |              10 |
+|                | `*`                        |           5 |             150 |
+
+### Growth Plan Rate Limits
+
+| Environment    | Path / Bucket              | Burst (rps) | Sustained (rpm) |
+| :------------- | :------------------------- | ----------: | --------------: |
+| **Production** | `/sessions/whoami`         |         800 |           18000 |
+|                | `/admin/oauth2/introspect` |         800 |           18000 |
+|                | `/relation-tuples/check`   |         800 |           18000 |
+|                | `GET /admin/identities`    |          20 |             600 |
+|                | `*`                        |         400 |            9000 |
+| **Staging**    | `/sessions/whoami`         |          10 |              30 |
+|                | `/admin/oauth2/introspect` |          10 |             300 |
+|                | `/relation-tuples/check`   |          10 |             300 |
+|                | `GET /admin/identities`    |           1 |              10 |
+|                | `*`                        |           5 |             150 |
+
+## Important Notes on Project-Based Rate Limits
+
+### Workspace Rate Limit
+
+For all projects in a workspace, the rate limit is the same as for the production environment of your plan. For example, on the
+Production plan, the rate limit would be 80 rps on the `/sessions/whoami` path for all projects combined in the workspace.
```

Comment:
Are you sure about that? Clients had a single workspace with both prod and dev projects, and they had ratelimit keys based only on the project tier. Our envoy based config has no information about workspaces, only project data

Reply:
I copied the current docs on the project tier-based rate-limits with a todo to check with you and Patrik. I will do so this week.